plugin/nvim-treesitter.vim

```diff
@@ -41,6 +41,7 @@ highlight default link TSProperty Identifier
 highlight default link TSConstructor Special
 highlight default link TSAnnotation PreProc
 highlight default link TSAttribute PreProc
+highlight default link TSNamespace Constant
```

Comment:
As @undu suggested, it sounds better to have that highlighted as `Include` instead of `Constant`.

Reply:
tested the last 24h with different color schemes. `Include` looks generally good although it is close to functions in some color schemes.